src/appleseed/renderer/modeling/camera/thinlenscamera.cpp

```diff
@@ -750,21 +750,15 @@ DictionaryArray ThinLensCameraFactory::get_input_metadata() const
             .insert("name", "focal_distance")
             .insert("label", "Focal Distance")
             .insert("type", "text")
-            .insert("use", "optional")
-            .insert("visible_if",
-                Dictionary()
-                    .insert("autofocus_enabled", "false")));
+            .insert("use", "optional"));
```

Comment:
Maybe focal_distance can have default value, compared with `autofocus_target`?

Reply:
I was thinking about that. The default value would be empty, correct? I think empty is the default value for default values :)

Reply:
Sorry, I can't understand what you saying. Poor English :(

Reply:
No problem, your English is fine.

You said:

> Maybe focal_distance can have default value, compared with autofocus_target?

Maybe I misunderstood you. What did you have in mind?

Reply:
Oh I just wondering why `focal_distance` has no default value, not like `autofocus_target`.

Reply:
Ok, that's what I understood.

The default value of `focal_distance` should be `""` (empty string), do you agree? In that case, the empty string is already the default value (I think) so there's no need to specify it. But you can add it if you want, maybe it would make things clearer.

Reply:
In my understand, both `m_focal_distance` and `m_autofocus_target` is set to some value, whether they are defined or not in edit window (in `extract_focal_distance()`), and I guess `.insert("default", "0.0")` make text that shows in edit window, when they are not defined in project file.

..So what I intend is, as you said, I'd like insert default value same with `DefaultFocalDistance`. Then I'll commit it soon.

Reply:
Oh, I think I understand what you mean. Ok then, go ahead with adding a default value for `focal_distance`. 1.0 is a good default value.